### PR TITLE
Added Kuwait holidays

### PR DESCRIPTION
**Pull request by proghasan**
Add Kuwait Public Holidays.

### CODE REVIEW

File: src/Countries/Kuwait.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            'New Year\'s Day' => '01-01',
+            'National Day' => '02-25',
+            'Liberation Day' => '02-26',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = CarbonImmutable::createFromTimestamp(easter_date($year))
```
Comment (Nielsvanpach) [inline src/Countries/Kuwait.php:26]: Can be removed if Easter is not relevant for Kuwait

File: src/Countries/Kuwait.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            ->setTimezone('Asia/Kuwait');
+
+        // If there are any specific variable holidays for Kuwait, you can add them here.
+        return [];
```
Comment (Nielsvanpach) [inline src/Countries/Kuwait.php:30]: TODO

**Review by Nielsvanpach — CHANGES_REQUESTED** [review]

File: src/Countries/Kuwait.php (hunk 1 above)
Comment (proghasan) [reply src/Countries/Kuwait.php]: I removed easter from Kuwait.

**Comment by proghasan** [comment]
Hey @Nielsvanpach, 
Most of the errors come from another country's test case. Could you help me, how to solve those issues?

File: src/Countries/Kuwait.php (hunk 2 above)
Comment (proghasan) [reply src/Countries/Kuwait.php:27]: Update comment

**Comment by Nielsvanpach** [comment]
You should rebase your branch on this repo's main branch.

I'm not able to merge, as this feels too limited. I don't think Kuwait only has 3 public holidays. You can look at other Islamic countries to see how they managed Islamic holidays (Turkey for example)

**Pull request closed without merging**